### ISSUE TITLE
Cannot install 404 Not found

### ISSUE THREAD

**Issue by VaiTon** [flathub/com.anydesk.Anydesk#41]
When installing via `flatpak install flathub com.anydesk.Anydesk` i get:
```
Error: While downloading https://download.anydesk.com/linux/anydesk-5.5.5-amd64.tar.gz: Server returned status 404: Not Found
error: Failed to install com.anydesk.Anydesk: While downloading https://download.anydesk.com/linux/anydesk-5.5.5-amd64.tar.gz: Server returned status 404: Not Found
```

OS: Kubuntu 20.04
Flatpak: 1.6.3

**Comment by nazar2sfive** [comment]
```
        ID                           Branch          Op         Remote          Download
 1. [✗] com.anydesk.Anydesk          stable          i          flathub         1.0 kB / 7.3 MB

Error: While downloading https://download.anydesk.com/linux/anydesk-5.5.5-amd64.tar.gz: Server returned status 404: Not Found
error: Failed to install com.anydesk.Anydesk: While downloading https://download.anydesk.com/linux/anydesk-5.5.5-amd64.tar.gz: Server returned status 404: Not Found

```

**Comment by VaiTon** [comment]
Should be fixed by #40.
